components/index.js

```diff
@@ -20,9 +20,9 @@ export { default as FormToggle } from './form-toggle';
 export { default as FormTokenField } from './form-token-field';
 export { default as IconButton } from './icon-button';
 export { default as KeyboardShortcuts } from './keyboard-shortcuts';
-export { default as MenuItemsChoice } from './menu-items/menu-items-choice';
-export { default as MenuItemsGroup } from './menu-items/menu-items-group';
-export { default as MenuItemsToggle } from './menu-items/menu-items-toggle';
+export { default as MenuItemsChoice } from './menu-items/choice';
+export { default as MenuItemsGroup } from './menu-items/group';
+export { default as MenuItemsItem } from './menu-items/item';
```

Comment:
@gziolo This one is a mouthful to expose like this. Can we simplify to `Menu` and `MenuItem`?

Reply:
`MenuItem` sounds good, but `Menu` doesn't fit really because you can have a few groups inside one menu.

Reply:
Done in #5798.